Doc/library/tkinter.commondialog.rst

```diff
@@ -0,0 +1,25 @@
+:mod:`tkinter.commondialog` --- Standard dialog windows
+=======================================================
+
+.. module:: tkinter.commondialog
+   :platform: Tk
+   :synopsis: Tkinter base class for dialogs
+
+**Source code:** :source:`Lib/tkinter/commondialog.py`
+
+--------------
+
+The :mod:`tkinter.commondialog` module provides the :class:`Dialog` class that
+is the base class for dialogs defined in other supporting modules.
+
+.. class:: Dialog(master=None, **options)
+
+   .. method:: show(color=None, **options)
+
+      The *show* method renders the Dialog window.
```

Comment:
The current standard is for docstrings and the doc entry to start with an active verb.  In particular, 'Return ..' rather than 'Returns ...'.  Lots of changes.  Here, 'The show method' is redundant.  Condense to

"Render the Dialog window."